### DISCUSSION TITLE
Help in understanding negative predictions

### DISCUSSION THREAD

**Discussion by pbanjara-debug** [neuralhydrology/neuralhydrology#277] [Q&A]
I am using NeuralHydrology to predict reservoir inflows using a combination of static attributes (e.g., degree of regulation) and dynamic meteorological variables (precipitation, Tmax, Tmin, wind). My training process looks normal the training and validation loss decreases throughout the epochs. However, during testing of out-of-sample experiments, I notice that some basins produce negative inflow predictions for the entire time series.

What might be causing this issue, and how can I debug it? Any guidance would be greatly appreciated.
